README.md

```diff
@@ -2,7 +2,7 @@
 
 The OCI Image Format project creates and maintains the software shipping container image format spec (OCI Image Format). The goal of this specification is to enable the creation of interoperable tools for building, transporting, and preparing a container image to run.
 
-This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a [filesystem serialization](serialization.md), and an [image configuration](serialization.md#image-json-description). At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more filesystem serialization archives that will be unpacked to make up the final runnable filesystem. The image configuration includes information such as application arguments, environments, etc. The combination of the image manifest, image configuration, and one or more filesystem serializations is called the "OCI Image".
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a [filesystem serialization](config.md), and an [image configuration](config.md#image-json-description). At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more filesystem serialization archives that will be unpacked to make up the final runnable filesystem. The image configuration includes information such as application arguments, environments, etc. The combination of the image manifest, image configuration, and one or more filesystem serializations is called the "OCI Image".
```

Comment:
I'd rather handle the `serialization.md` → `config.md` rename in a separate commit, instead of making it an unrelated side-effect of cd62032f (layer: add introductory comments, 2016-08-31).  But either way, if we're handling the rename in this PR I think we want “filesystem serialization” → “image configuration” too.  All of these JSON documents are serializing some portion of the image.  The thing that's unique to application/vnd.oci.image.config.v1+json is that it also contains `User`, `Memory`, etc.